statsd-client-async/jbuild

```diff
@@ -3,4 +3,5 @@
 (library
  ((name statsd_client_async)
   (public_name statsd-client-async)
-  (libraries (async))))
\ No newline at end of file
+  (preprocess (pps (ppx_sexp_conv)))
+  (libraries (sexplib async))))
```

Comment:
It would be nice to alphabetize library names where possible, but it's not a huge deal.

Reply:
Good idea. An old coworker had the opinion that almost everything should be alphabetized, unless it has an extremely obvious or semantically important alternative ordering. I thought that was a bit radical at first, but after a while, it really grew on me.